scripts/create-pr-1480.ps1

```
@@ -0,0 +1,100 @@
+#   1. -Token <PAT>            (Personal Access Token)
+#   2. $env:GITHUB_TOKEN
+#   3. $env:GH_TOKEN
+#   4. `gh auth token`         (if gh CLI is installed)
+#   5. none                    -> falls back to printing the compare URL
+#
+# Examples:
+#   powershell -ExecutionPolicy Bypass -File scripts\create-pr-1480.ps1
+#   powershell -ExecutionPolicy Bypass -File scripts\create-pr-1480.ps1 -Token ghp_xxxx -OpenBrowser
+
+param(
+    [string]$Token = "",
```

PSScriptAnalyzer (1.25.0)

[warning] The parameter 'Token' has been declared but not used.

(PSReviewUnusedParameter)